github-custom-fonts.user.css

```
@@ -1,7 +1,7 @@
 /* ==UserStyle==
 @name         GitHub Custom Fonts
 @namespace    StylishThemes
-@version      5.2.13
+@version      5.2.14
```

```suggestion
@version      5.2.13
```

don't alter version, this will be done automatically during the release process